cpp/src/fil/treelite_import.cu

```diff
@@ -674,6 +680,8 @@ void from_treelite(const raft::handle_t& handle,
       "doesn't support inferencing models with float64 values. "
       "This may lead to predictions with reduced accuracy.");
   }
+  // same as std::common_type: float+double=double, float+int64_t=float
+  typedef decltype(threshold_t{} + leaf_t{}) real_t;
```

Comment:
Let's use `using` where possible to avoid having to think about if a type alias will work for all template applications.

Reply:
Done.